cfgov/v1/tests/test_forms.py

```diff
@@ -9,16 +9,6 @@
 
 class TestFilterableListForm(TestCase):
 
-    @mock.patch('v1.forms.FilterableListForm.__init__')
-    @mock.patch('taggit.models.Tag.objects')
-    def test_set_topics_filters_tags_on_pageids(self, mock_tag_objects, mock_init):
```

Comment:
I'm working on writing a replacement for this test. This logic no longer lives in the FilterableListForm, and it should have a test that depends less on implementation anyway.